mybatis-plus/src/main/java/com/baeldung/mybatisplus/migration/DBMigration.java

```diff
@@ -0,0 +1,22 @@
+package com.baeldung.mybatisplus.migration;
+
+import java.util.Collections;
+import java.util.List;
+
+import org.springframework.stereotype.Component;
+
+import com.baomidou.mybatisplus.extension.ddl.SimpleDdl;
+
+@Component
+public class DBMigration extends SimpleDdl {
+
+    @Override
+    public List<String> getSqlFiles() {
+
+        // works for MySQL setup
+        //return Arrays.asList("db/db_v1.sql", "db/db_v2.sql");
+
+        return Collections.emptyList();
```

Comment:
This was not addressed

Reply:
I've changed it to `Arrays.asList("db/db_v1.sql", "db/db_v2.sql")`. However, DDL migration don't work for H2 - only MySQL, Postgres etc